Examples/River_area_test/MainMeshGenerationKratos.py

```diff
@@ -0,0 +1,73 @@
+import shapefile    
+import gmsh
+import sys
+import meshio
+
+
+# importing external classes in order to build tridimensional terrain mesh automatycally 
+from contour_input_output import ContourInputOutput
+from gmsh_contour_2D_mesh import GmshContour2dMeshGenerator
+# from simulation_mesh_medpa import SimulationMeshMdpa
+from gmsh_3d_mesh_terrain import Gmsh3dMeshTerrain
+from kratos_terrain_elevation import TerrainElevationFromRaster
+from simulation_mesh_mdpa import SimulationMeshMdpa
+import KratosMultiphysics 
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+#from KratosMultiphysics.CoSimulationApplication.utilities import model_part_utilities
```

Comment:
```suggestion
#from KratosMultiphysics.CoSimulationApplication.utilities import model_part_utilities
```
I think it can be removed.